src/main/scala/gestalt/dotty/Expander.scala

```diff
@@ -67,7 +76,8 @@ object Expander {
   /** Expand def macros */
   def expandDefMacro(tree: tpd.Tree)(implicit ctx: Context): untpd.Tree = tree match {
     case ExtractApply(Select(obj, method), targs, argss) =>
-      val className = obj.symbol.info.classSymbol.fullName + "$inline$"
+      val objType = obj.symbol.info.resultType
+      val className = javaClassName(objType.classSymbol) + "$inline$"
```

Comment:
`resultType` here seems a little strange. We usually use that for method types or poly types. Does `obj.symbol.info.classSymbol` do the same?